pages/common/ssh-add.md

```diff
@@ -0,0 +1,23 @@
+# ssh-add
+
+> Manage loaded RSA or DSA keys in the ssh-agent.
```

Comment:
It can manage ed25519 keys too. So I believe the term "ssh keys" should be sufficient.

Also add a second line as a subheading that "Ensure that `ssh-agent` is up and running for the keys to be loaded in it."

Reply:
Done, wasn't too sure about the wording myself, since the documentation usually talks about identities instead of keys.